frontend/mr-admin-flate/src/pages/TiltaksgjennomforingPage.tsx

```diff
@@ -1,8 +1,14 @@
-import { Alert } from "@navikt/ds-react";
-import { Link } from "react-router-dom";
+import { Alert, Heading } from "@navikt/ds-react";
 import { useTiltaksgjennomforingById } from "../api/tiltaksgjennomforing/useTiltaksgjennomforingById";
+import Tilbakeknapp from "mulighetsrommet-veileder-flate/src/components/tilbakeknapp/Tilbakeknapp";
 
-export function TiltaksgjennomforingPage() {
+interface TiltaksgjennomforingPageProps {
+  fagansvarlig?: boolean;
+}
+
+export function TiltaksgjennomforingPage({
+  fagansvarlig = false,
+}: TiltaksgjennomforingPageProps) {
```

Comment:
Lurer på om det er bedre å lage en egen komponent for tiltaksgjennomføringer for fagansvarlige istedenfor å ta inn et flagg (fagansvarlig-prop'en). Det blir ganske uryddig og rotete på sikt dersom vi skal styre komponentene basert på boolske verdier, istedenfor å ha konkrete komponenter for det.